docs/contributing/mautic_style_guide.rst

```diff
@@ -1,2 +1,187 @@
-Mautic Style Guide
-##################
\ No newline at end of file
+Mautic style guide
+##################
+The Mautic Style guide recommends best practices that aims to bring about a consistent style, voice, and tone across the documentation. 
+
+Vale
+****
+Vale is a natural language linter that supports plain text, markup (Markdown, reStructuredText, AsciiDoc, and HTML), and source code comments. Vale doesn't attempt to offer a one-size-fits-all collection of rules—instead, it strives to make customization as easy as possible.
+
+How to install vale
+===================
+Before pushing, run Vale and address suggestions and errors as applicable.
+
+#. Install vale
```

Comment:
You need to add this link https://docs.errata.ai/vale/install to Vale, so its easier for users